mod/01_core/set/all/fetch.rb

```diff
@@ -54,8 +54,11 @@ def fetch mark, opts={}
     if card.new_card?
       if opts[:new]
         return card.renew(opts) if !clean_cache_opts? opts
+      elsif opts[:skip_virtual]
```

Comment:
Took me a while to parse that.  You're saying that Card.fetch mark is the same as the hypothetical known I was describing?  I like the get idea.  we could probably use that for a lot of calls.